Add crontab to refresh repos

carlodx opened the pull request.

Commit:
Add crontab to refresh repos every 5 mins

nathankleyn reviewed (changes requested):
Spoke with @carlosdiazh but will need a slight tweak to allow it to work with the upstream image as the upstream image calls `entrypoint.sh` expecting it not to block.

We agreed to break out the `crond` call to another place so we can background it when we run the other image. 👍

nathankleyn reviewed (commented):
LGTM 👍

nathankleyn committed:
EVE-2075 Use `exec` when calling `crond` to make `SIGKILL` work.

Following the guidelines from https://docs.docker.com/engine/reference/builder/#shell-form-entrypoint-example, use `exec` when calling `crond -f` to make sure `SIGKILL` actually works when `docker stop` runs.

nathankleyn reviewed (approved):
LGTM 👍
